src/autogluon_dashboard/plotting/interactive_df.py

```diff
@@ -1,26 +1,34 @@
+from typing import Optional
+
 import hvplot
 import pandas
 
-from ..scripts.utils import get_df_filter_by_framework
+from ..scripts.utils import get_df_filter_by_dataset, get_df_filter_by_framework
 from .all_plots import Plot
 
 
 class InteractiveDataframe(Plot):
     def __init__(
-        self,
-        df_process: hvplot.Interactive,
-        framework: str,
-        width: int,
+        self, df_process: hvplot.Interactive, framework: str, width: int, dataset: Optional[str] = None
     ) -> None:
         self.dataset_to_plot = df_process
         self.framework = framework
         self.table_width = width
+        self.dataset = dataset
 
     def get_interactive_df(self) -> hvplot.Interactive:
-        return hvplot.bind(self._preprocess, self.framework).interactive(width=self.table_width)
+        return hvplot.bind(self._preprocess, self.framework, self.dataset).interactive(width=self.table_width)
 
     def _preprocess(*args) -> pandas.DataFrame:
```

Comment:
Probably missed this in previous PR. But would recommend changing to kwargs instead of args. Can be addressed in separate PR